site/mkdocs.yml

```diff
@@ -49,8 +49,9 @@ nav:
   - User docs:
     - Getting Started: getting-started.md
     - Spark: spark.md
-    - Spark Structured Streaming: spark-structured-streaming.md
+    - Spark Streaming: spark-structured-streaming.md
     - Presto: presto.md
+    - Maintenance: maintenance.md
```

Comment:
Looks like the page is missing in this PR - probably miss to add a new file?

Reply:
Fixed.